### PR TITLE
Bugfix/plan time

### PR DESCRIPTION
**Pull request by kcoleman731**
*Issue #, if available:*

*Description of changes:*

Optimizing TF plan time. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by askulkarni2 — APPROVED** [review]

File: modules/irsa/data.tf [hunk 1]
```diff
@@ -23,21 +23,13 @@ data "aws_iam_policy_document" "irsa_with_oidc" {
...
     principals {
       type        = "Federated"
-      identifiers = [local.eks_oidc_provider_arn]
+      identifiers = [var.eks_oidc_provider_arn]
     }
 
     condition {
       test     = "StringEquals"
-      variable = "${local.eks_oidc_issuer_url}:sub"
+      variable = "${var.eks_oidc_issuer_url}:sub"
```
Comment (vara-bonthu) [inline modules/irsa/data.tf:31]: This should be `variable = "${replace(var.eks_oidc_issuer_url, "https://", "")}:sub"`

**Review by vara-bonthu — CHANGES_REQUESTED** [review]
One change required

File: modules/irsa/outputs.tf [hunk 2]
```diff
@@ -18,10 +18,10 @@
 
 output "irsa_iam_role_arn" {
   description = "IAM role ARN for your service account"
-  value       = aws_iam_role.irsa.arn
+  value       = ""
```
Comment (vara-bonthu) [inline modules/irsa/outputs.tf:21]: Is this correct for output variable?

File: modules/irsa/outputs.tf [hunk 3]
```diff
@@ -18,10 +18,10 @@
...
 }
 
 output "irsa_iam_role_name" {
   description = "IAM role name for your service account"
-  value       = aws_iam_role.irsa.name
+  value       = ""
```
Comment (vara-bonthu) [inline modules/irsa/outputs.tf:26]: same as above

**Review by vara-bonthu — CHANGES_REQUESTED** [review]
I think we figured out the issue is due to `depends_on` on module causing the delays. Do we still need this approach? Just to reduce passing too many variables for each add-on.

**Pull request closed without merging**